### PR TITLE
Cleanup/remove unchanged packages

### PR DESCRIPTION
**Pull request by Ivo-Yankov**
**Description**:
Remove the following packages:

- abi
- asm
- base64
- basex
- keccak256
- pbkdf2
- properties
- random
- rlp
- sha2
- solidity
- web
- wordlists
- hardware-wallet
- experimental

Changes made to the following packages were either irrelevant or reverted and the packages are removed:

- bytes
- shims
- signing-key
- hash
- BigNumber

Rename the `ethers` package to `hethers`

Change the `name` field in `packages/*/package.json` to: `@hethers/{packageName}` and the version to `1.0.0`

Change the `name` field in `./package.json` to: `@hashgraph/hethers` and the version to `1.0.0`

**Related issue(s)**:

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: misc/admin/lib/cmds/hoist.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ const log_1 = require("../log");
         const dependencies = (0, local_1.getDependencies)(null, (name) => {
             return !(0, path_1.isEthers)(name);
         });
+        console.log(dependencies);
```
Comment (Daniel-K-Ivanov) [inline misc/admin/lib/cmds/hoist.js:20]: nit: unnecessary change

File: package.json [hunk 2]
```diff
@@ -75,24 +74,32 @@
...
   },
   "dependencies": {
-    "@babel/parser": "7.8.4",
-    "@babel/types": "7.8.3",
+    "@ethersproject/abi": "5.5.0",
+    "@ethersproject/base64": "5.5.0",
+    "@ethersproject/basex": "5.5.0",
+    "@ethersproject/bignumber": "5.5.0",
+    "@ethersproject/bytes": "5.5.0",
+    "@ethersproject/cli": "5.5.0",
```
Comment (Daniel-K-Ivanov) [inline package.json:82]: We should not need to include the `cli` package

File: package.json [hunk 3]
```diff
@@ -75,24 +74,32 @@
...
+    "@ethersproject/hash": "5.5.0",
+    "@ethersproject/keccak256": "5.5.0",
+    "@ethersproject/pbkdf2": "5.5.0",
+    "@ethersproject/properties": "5.5.0",
+    "@ethersproject/random": "5.5.0",
+    "@ethersproject/rlp": "5.5.0",
```
Comment (Daniel-K-Ivanov) [inline package.json:88]: We should not need to include the `rlp` package

File: packages/address/package.json [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
-    "@ethersproject/bignumber": "^5.5.0",
-    "@ethersproject/bytes": "^5.5.0",
-    "@ethersproject/keccak256": "^5.5.0",
-    "@ethersproject/logger": "^5.5.0",
-    "@ethersproject/rlp": "^5.5.0"
+    "@ethersproject/bignumber": "5.5.0",
+    "@ethersproject/bytes": "5.5.0",
+    "@ethersproject/keccak256": "5.5.0",
+    "@hethers/logger": "1.0.0",
+    "@ethersproject/rlp": "5.5.0"
```
Comment (Daniel-K-Ivanov) [inline packages/address/package.json:7]: I think that we do not need the `rlp` package

File: packages/hethers/package.json [hunk 5]
```diff
@@ -1,34 +1,34 @@
...
+    "@hethers/contracts": "1.0.0",
     "@ethersproject/hash": "5.5.0",
-    "@ethersproject/hdnode": "5.5.0",
-    "@ethersproject/json-wallets": "5.5.0",
+    "@hethers/hdnode": "1.0.0",
+    "@hethers/json-wallets": "1.0.0",
+    "@hethers/logger": "1.0.0",
+    "@hethers/networks": "1.0.0",
+    "@hethers/providers": "1.0.0",
+    "@ethersproject/rlp": "5.5.0",
```
Comment (Daniel-K-Ivanov) [inline packages/hethers/package.json]: Maybe we should remove the `rlp` package since we do not need it

File: packages/providers/package.json [hunk 6]
```diff
@@ -13,23 +13,23 @@
...
+    "@ethersproject/basex": "5.5.0",
+    "@ethersproject/bignumber": "5.5.0",
+    "@ethersproject/bytes": "5.5.0",
+    "@hethers/constants": "1.0.0",
+    "@ethersproject/hash": "5.5.0",
+    "@hethers/logger": "1.0.0",
+    "@hethers/networks": "1.0.0",
+    "@ethersproject/properties": "5.5.0",
+    "@ethersproject/random": "5.5.0",
+    "@ethersproject/rlp": "5.5.0",
```
Comment (Daniel-K-Ivanov) [inline packages/providers/package.json]: We do not need the rlp package. Maybe we should remove it

File: packages/tests/package.json [hunk 7]
```diff
@@ -1,9 +1,9 @@
 {
   "dependencies": {
-    "@ethersproject/cli": "^5.5.0",
-    "@ethersproject/testcases": "^5.5.0",
+    "@ethersproject/cli": "5.5.0",
```
Comment (Daniel-K-Ivanov) [inline packages/tests/package.json]: We shouldn't need the `cli` as dependency

File: packages/transactions/package.json [hunk 8]
```diff
@@ -1,14 +1,14 @@
...
-    "@ethersproject/rlp": "^5.5.0",
-    "@ethersproject/signing-key": "^5.5.0",
+    "@hethers/address": "1.0.0",
+    "@ethersproject/bignumber": "5.5.0",
+    "@ethersproject/bytes": "5.5.0",
+    "@hethers/constants": "1.0.0",
+    "@ethersproject/keccak256": "5.5.0",
+    "@hethers/logger": "1.0.0",
+    "@ethersproject/properties": "5.5.0",
+    "@ethersproject/rlp": "5.5.0",
```
Comment (Daniel-K-Ivanov) [inline packages/transactions/package.json]: We should remove the `rlp` package

**Review by Daniel-K-Ivanov — CHANGES_REQUESTED** [review]
Good job! 🚀

File: misc/admin/lib/cmds/hoist.js (hunk 1 above)
Comment (Ivo-Yankov) [reply misc/admin/lib/cmds/hoist.js]: removed it

File: package.json (hunk 3 above)
Comment (Ivo-Yankov) [reply package.json]: `rlp` package is removed

File: packages/address/package.json (hunk 4 above)
Comment (Ivo-Yankov) [reply packages/address/package.json]: `rlp` package is removed

File: packages/hethers/package.json (hunk 5 above)
Comment (Ivo-Yankov) [reply packages/hethers/package.json]: `rlp` package is removed

File: packages/providers/package.json (hunk 6 above)
Comment (Ivo-Yankov) [reply packages/providers/package.json]: `rlp` package is removed

File: packages/tests/package.json (hunk 7 above)
Comment (Ivo-Yankov) [reply packages/tests/package.json]: `cli` package is removed

File: packages/transactions/package.json (hunk 8 above)
Comment (Ivo-Yankov) [reply packages/transactions/package.json]: `rlp` package is removed

**Review by Daniel-K-Ivanov — APPROVED** [review]
